### PR TITLE
Fix(button): fix button styles

### PR DESCRIPTION
**Pull request by ceciliamvrie**
Modified: 
* Button

Fixed button text color to not match the background color
of the raised and flat buttons. 

Issues: none

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -12,3 +12,4 @@
  Pete Givens <pgivens@gmail.com>
  Julianne Johnson <tweetypie417@yahoo.com>
  calebjanderson <caleb.anderson130@gmail.com>
+ ceciliamvrie <ceciliamvrie@gmail.com>
```
Comment (martahj) [inline AUTHORS]: Is your UT email still valid? If not, you might want to remove it from the authorslist

**Comment by martahj** [comment]
Looks good! Could you do the following:
* generate a link to a Now app (you can do this by running `yarn now-build` from the root of the smc directory) and post the link so we can check out your work?
* Bump the version by running `npm version patch` from the same directory

Thanks!

**Comment by ceciliamvrie** [comment]
https://docs-plkeldbnkl.now.sh/chips/ here's the link!

File: AUTHORS (hunk 1 above)
Comment (ceciliamvrie) [reply AUTHORS]: Ill make sure if it's still valid

**Comment by ceciliamvrie** [comment]
https://docs-fmvozljlng.now.sh

**Pull request merged**